app/user.go

```diff
@@ -980,7 +980,8 @@ func (a *App) invalidateUserChannelMembersCaches(userId string) *model.AppError
 }
 
 func (a *App) UpdateActive(user *model.User, active bool) (*model.User, *model.AppError) {
-	user.UpdateAt = model.GetMillis()
+	updateTime := model.GetMillis()
+	user.UpdateAt = updateTime
```

Comment:
After the previous changes, it looks declaring a separate `updateTime` is no longer required.